docs/algorithms/castor/special-features/hvdc.md

```diff
@@ -0,0 +1,110 @@
+# HVDC
+
+## Definition
+
+HVDC lines are modeled using [HvdcLine of the IIDM PowSyBl network model](inv:powsyblcore:*:*#hvdc-line), and an HVDC Range Action will always point to an HVDC line.
+
+An HVDC line can operate in **two modes**:
+
+1. **Fixed Setpoint Mode**  
+   In this mode, the power flow on the HVDC line is directly controlled via its **active power setpoint**.
+
+2. **AC Emulation Mode**  
+   The HVDC line operates in AC Emulation mode when the extension [**angle-droop active power control**](inv:powsyblecore:*:*#hvdc-angle-droop-active-power-control) is enabled. In this mode, the flow is determined by the **phase difference**:  
```

Comment:
For some reason, I cannot see the [**angle-droop active power control**](inv:powsyblecore:*:*#hvdc-angle-droop-active-power-control) on the website version, I only have: 
"The HVDC line operates in AC Emulation mode when the extension is enabled."
Can you check if it is also the case for you?